Makefile

```diff
@@ -17,7 +17,6 @@ buf: bufsetup
 
 clean:
 	rm -rf grpc
-	rm -rf bin
```

Comment:
This fix should have been in the other PR but it is still important to have. `make clean` should clear the built go code.

Reply:
Done